packages/core/src/services/ubi/community/details.ts

```diff
@@ -519,8 +512,7 @@ export class CommunityDetailsService {
         let beneficiariesSubgraph: BeneficiarySubgraph[] | null = null;
 
         if (filter.state !== undefined) {
-            beneficiaryState =
-                filter.state === 'active' ? 'state: 0' : 'state: 1';
+            beneficiaryState = `state: ${filter.state ? filter.state : 0}`;
```

Comment:
little trick
```suggestion
            beneficiaryState = `state: ${filter.state || 0}`;
```
this can apply to a couple others here.